spec/services/FetchMoviesSpec.php

```diff
@@ -6,24 +6,23 @@
 
 describe(FetchMovies::class, function () {
 
-    describe("->getMovies()", function () {
+    describe("->getLatestMovie()", function () {
         it('fetches a json string', function () {
             $fetcher = new FetchMovies();
             expect(FetchMovies::class)
             ->toReceive('getJsonString');
-            $fetcher->getMovies();
         });
 
         it('converts the json string into an array', function () {
             $fetcher = new FetchMovies();
-            $movieDataArr = $fetcher->getMovies();
+            $movieDataArr = $fetcher->getLatestMovie();
```

Comment:
following the convention here, in the future, variables are underscored such that `$movieDataArr` should be `$movie_data_arr`. Nobiggie but start practicing.

Reply:
cool should I refactor the var names in this scope or just continue in the future with this naming structure?